src/libkomwm.py

```diff
@@ -505,6 +514,7 @@ def addPattern(dashes):
         if int(row[5]) < cnt:
             raise Exception('Wrong type id: {0}'.format(';'.join(row)))
         while int(row[5]) > cnt:
+            # TODO: Why do we have those lines in `types.txt`? What if remove print(...) statement?
             print("mapswithme", file=types_file)
```

Comment:
Looks like order of lines in `types.txt` matters. And we add `mapswithme` lines for missing types.

Is it correct?

Reply:
Yeap order is important and is the same as type's ID

Reply:
Ok, removed TODO comment